cmd/modern/root/install/mssql-base.go

```diff
@@ -603,7 +603,7 @@ func (c *MssqlBase) downloadImage(
 	output *output.Output,
 	controller *container.Controller,
 ) {
-	output.Infof(localizer.Sprintf("Downloading %v", imageName))
+	output.Info(localizer.Sprintf("Downloading %v", imageName))
 	err := controller.EnsureImage(imageName)
 	if err != nil || c.unitTesting {
 		output.FatalfErrorWithHints(
```

Comment:
>.FatalfErrorWithHints [](http://example.com/codeflow?start=8&length=21)

this call looks problematic as well

Reply:
Yes. Replaced the call with `FatalErrorWithHints`